### PR TITLE
allow jxcore workable

### PR DESCRIPTION
**Pull request by darrrk**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -86,8 +86,9 @@ function writeConfig (fd, flags, cb) {
 
 module.exports = function (cb) {
   // bail early if this is not node
-  var isElectron = process.versions && process.versions.electron;
-  if (isElectron) {
+  var isElectron = process.versions && process.versions.electron,
+    isJXcoreSM = process.versions && process.versions.jxcore && process.versions.sm;
```
Comment (phated) [inline index.js]: each variable should be defined with its own `var`

**Comment by phated** [comment]
@darrrk it looks like https://github.com/jxcore/jxcore/issues/551#issuecomment-141131278 fixes the issue?  Can you confirm?

**Comment by darrrk** [comment]
@phated, it seems you are right.

**Comment by darrrk** [comment]
May reopen, if https://github.com/jxcore/jxcore/commit/7362b14ec29ec9cc9eedcb626b496dfece786ba8 not working properly

**Pull request closed without merging**